exercises/series/src/example/java/Series.java

```diff
@@ -4,21 +4,20 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class Series {
+class Series {
 
     private final int digitsSize;
     private List<Integer> digits;
 
-    public Series(String string) {
-        this.digits = Arrays
-                .asList(string.split(("")))
-                .stream()
-                .map(digit -> Integer.parseInt(digit))
-                .collect(Collectors.toList());
+    Series(String string) {
+        this.digits =
+                Arrays.stream(string.split(("")))
```

Comment:
The defaults in IntelliJ (and probably also Eclipse) should be fine for formatting :) The [google style guide formatting section](https://google.github.io/styleguide/javaguide.html#s4-formatting) would probably be useful if you want something for reference. I've mostly learnt best practices from experience and code reviews, so I haven't read the google style guide cover to cover, but it looks sensible and I think many people use it :)